### PR TITLE
TOURCMS-3107 ListOrders - Build new booking search endpoint

### PR DESCRIPTION
**Pull request by MiguelMachadoPalisis**
Added new endpoint api_cp_list_bookings

    Available parameters in query:

    Filter by date:
     - made_date_start
     - made_date_end
     - start_date_start
     - start_date_end
     - cancel_date_start
     - cancel_date_end
     - end_date_start
     - end_date_end

    Filter by channel_id_tour_id parameter

    Filter by agent_customer_reference parameter

### CODE REVIEW

File: src/TourCMS.php [hunk 1]
```diff
@@ -392,6 +392,14 @@ public function search_bookings($params = "", $channel = 0)
 			return($this->request('/c/bookings/search.xml?'.$params, $channel));
 	}
 
+    public function list_bookings($params = "", $channel = 0)
+    {
+        if($channel==0)
+            return($this->request('/p/booking/list.xml?'.$parmas));
```
Comment (paulslugocki) [inline src/TourCMS.php:398]:
```suggestion
            return($this->request('/p/bookings/list.xml?'.$params));
```

File: src/TourCMS.php [hunk 2]
```diff
@@ -392,6 +392,14 @@ public function search_bookings($params = "", $channel = 0)
...
+        else
+            return($this->request('/c/booking/list.xmo?'.$params));
```
Comment (paulslugocki) [inline src/TourCMS.php:400]:
```suggestion
            return($this->request('/c/booking/list.xml?'.$params));
```

File: src/TourCMS.php (hunk 1 above)
Comment (MiguelMachadoPalisis) [reply src/TourCMS.php:398]: Updated, Thanks!!

File: src/TourCMS.php (hunk 2 above)
Comment (MiguelMachadoPalisis) [reply src/TourCMS.php:400]: Updated, Thanks!!

**Comment by paulslugocki** [comment]
@MiguelMachadoPalisis Please can you confirm you have tested this.

**Pull request merged**